class-wc-gateway-komoju.php

```diff
@@ -212,7 +212,7 @@ private function get_input_field_data()
         $komoju_client = $this->komoju_api;
 
         try {
-            $methods       = apply_filters( 'woocommerce_komoju_payment_methods', $komoju_client->paymentMethods());
+            $methods       = apply_filters('woocommerce_komoju_payment_methods', $komoju_client->paymentMethods());
```

Comment:
Lint problem from the PR made it into master. I forgot that Github doesn't run CI on PRs by default 😞

Reply:
@thoiberg Sounds like it should though. Maybe time to make another PR that enables lint on the CI? (No need to change this PR)

Reply:
Sorry I should be clearer 😄 We do run it on CI, but AFAIK Github won't run the actions automatically on public repos from non-project members (I believe too many people were abusing Actions to crypto mine 😒 ). I think project owners can kick off the builds but I'm not too sure on that.